pipeline/authn/authenticator_bearer_token.go

```diff
@@ -8,9 +8,11 @@ import (
 	"github.com/tidwall/gjson"
```

Comment:
Could we ensure in all the authenticators that the new TLS config is actually being used by adding tests for that case?